app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include <assert.h>
+#include <cassert>
```

Comment:
Do we need this change?

Reply:
It's not needed at all, I removed the `#include` altogether.